### PR TITLE
travis: Run dartfmt only on dev

### PR DESCRIPTION
**Pull request by kevmoo**
Run analyzer (at a minimum) across all packages at their oldest supported SDK

### CODE REVIEW

**Comment by jakemac53** [comment]
It looks like we need to add `deprecated_member_use_from_same_package: ignore` to the errors section of the analysis_options.yaml.

**Comment by kevmoo** [comment]
> It looks like we need to add `deprecated_member_use_from_same_package: ignore` to the errors section of the analysis_options.yaml.

Nope. Just run the right test...

File: pkgs/test/mono_pkg.yaml [hunk 1]
```diff
@@ -1,11 +1,15 @@
 dart:
- - dev
+  - dev
```
Comment (natebosch) [inline pkgs/test/mono_pkg.yaml]: Why was this section removed in `test_api` but not here?

Comment (kevmoo) [reply pkgs/test/mono_pkg.yaml]: Weirdness. `mono_repo` doesn't have support for specifying the SDK in a stage w/out a group – that's the short version.

**Pull request merged**